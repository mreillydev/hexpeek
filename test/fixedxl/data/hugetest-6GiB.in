# Overlapping replace, source < destination
FFFB3E10:100600000r@FFF37DDD:1000F0031
# Overlapping replace, source contained within destination
FFFA07E1:100711112r@FFFE92FF,22222
# Overlapping replace, source = destination
FFFFFFFF,3F918r@FFFFFFFF,11111
# Zero to zero replace
170F06000,20000r@170F08000,1001
# Overlapping replace, source > destination
170EDD000,37001r@170F00200,37001
# Deletion at start
0,1k
# Overlapping insertion, source < destination
1000i@0,20000
# Deletion at start
0,1k
# Deletion after middle
E0000000,40000k
# Overlapping insertion, source > destination
E0001000,300F7i@E0004100:E001B100
# Append
1800100f5,6000Fi99887700
# Print
0000000170f00000:-0
