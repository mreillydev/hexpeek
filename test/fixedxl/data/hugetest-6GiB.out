0000000170f00000: 00000000 00000000 00000000 00000000
*
0000000170f1cf00: 00000000 00000044 6688aacc ee113355
0000000170f1cf10: 7799bbdd ff002244 6688aacc ee113355
*
0000000170f240f0: 7799bbdd ff000000 00000000 00000000
0000000170f24100: 00000000 00000000 00000000 00000000
*
0000000170f40100: 00000000 00000044 6688aacc ee113355
0000000170f40110: 7799bbdd ff002244 6688aacc ee113355
*
00000001800100f0: 7799bbdd ff998877 00998877 00998877
0000000180010100: 00998877 00998877 00998877 00998877
*
0000000180070100: 00998877
